src/content/docs/accounts/accounts-billing/account-structure/new-relic-account-structure.mdx

```diff
@@ -9,26 +9,43 @@ redirects:
 
-At New Relic, an "organization" represents a New Relic customer. The organization contains everything relevant to a New Relic customer: its accounts, its users, and its data.
+When you sign up for New Relic, it creates an organization containing a single account. A Standard edition organization can only have a single account. Pro and Enterprise edition organizations can add more accounts. 
 
-When a New Relic organization is created, it contains a single account. A Standard edition organization can only have a single account, but Pro and Enterprise edition organizations can add more accounts. 
+An account can have many sources of data reporting to it. All data reported to New Relic requires an [account ID](/docs/accounts/accounts-billing/account-structure/account-id), which governs where that data goes and where it's stored. That ID is also used for some account-specific tasks, like making API calls. 
 
-Why an organization creates new accounts depends on their goals and structure. There's nothing preventing even quite large companies from having a single account, or a handful of accounts. The fewer accounts you have, the easier it is to see how all your monitored entities relate to each other using our platform. But there can be several reasons to add accounts, such as creating separate accounts for production and non-production environments, or if you want to establish firmer boundaries between different sets of data for any reason. If you're in doubt about why to create accounts, talk to your account representative.
+When you [set up user access](#account-access), you give users access to specific accounts. 
 
-Each account in an organization has its own [account ID](/docs/accounts/accounts-billing/account-setup/account-id), and that ID is used for some account-specific tasks, like making API calls.
+## Cross-account access [#cross-account-access]
 
-To add and rename accounts, see [Add accounts](/docs/accounts/accounts-billing/account-structure/add-accounts). 
+New Relic provides various cross-platform functionality that gives users access to data in multiple accounts. For example, you can assign users access to multiple accounts. And features like distributed tracing, custom dashboards, and workloads allow you to see data from entities monitored in other accounts, if you're assigned access to those accounts. 
 
-### How users access accounts [#account-access]
+But there can be some cross-account obstacles. For example, when using the [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder) to query your data, that will only query inside of the account you're in (to be able to make queries across accounts, you can use [our NerdGraph API](/docs/apis/nerdgraph/examples/nerdgraph-nrql-tutorial).) This is one reason to attempt to minimize the number of accounts you have. 
```

Comment:
I had to reread the second sentance of this paragraph a few times. Suggest splitting up or condensing.